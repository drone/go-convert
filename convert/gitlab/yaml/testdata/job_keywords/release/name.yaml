# https://docs.gitlab.com/ee/ci/yaml/#releasename

  release_job:
    stage: release
    release:
      name: 'Release $CI_COMMIT_TAG'