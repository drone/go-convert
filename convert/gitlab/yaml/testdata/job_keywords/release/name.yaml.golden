release_job:
    release:
        name: Release $CI_COMMIT_TAG
    stage: release
